Mathlib/Order/Cover.lean

```diff
@@ -245,10 +245,10 @@ theorem not_covBy [DenselyOrdered α] : ¬a ⋖ b := fun h =>
   h.2 hc.1 hc.2
 #align not_covby not_covBy
 
-theorem densely_ordered_iff_forall_not_covBy : DenselyOrdered α ↔ ∀ a b : α, ¬a ⋖ b :=
```

Comment:
You're renaming a theorem: can you add a deprecation warning and an allias, with a current date?
```suggestion
@[deprecated] alias densely_ordered_iff_forall_not_covBy := denselyOrdered_iff_forall_not_covBy -- 2024-04-04